infra/tpu-pytorch/infra_triggers.tf

```diff
@@ -7,4 +7,4 @@ module "terraform_apply" {
 
   worker_pool_id = module.worker_pool.id
   location       = "global"
-}
+}
```

Comment:
Is this change intentional?

Reply:
Seems like Manfei's commit accidentally got into the PR. @bhavya01 can you fix the rebase on this PR?